lib/std/c.zig

```diff
@@ -5703,6 +5703,9 @@ pub const SOCK = switch (native_os) {
         pub const RAW = 3;
         pub const SEQPACKET = 5;
         pub const MISC = 255;
+
+        pub const NONBLOCK = 0x40000;
+        pub const CLOEXEC = 0x80000;
```

Comment:
In my local Haiku tree, these seem to be:

```suggestion
        pub const CLOEXEC = 0x40;
        pub const NONBLOCK = 0x80;
```

Where did you find these values?

Reply:
/boot/system/develop/headers/posix/sys/socket.h